scipy/misc/_common.py

```diff
@@ -3,12 +3,19 @@
 (special, linalg)
 """
 
+from scipy._lib.deprecation import _deprecated
 from numpy import arange, newaxis, hstack, prod, array, frombuffer, load
 
+
 __all__ = ['central_diff_weights', 'derivative', 'ascent', 'face',
            'electrocardiogram']
 
 
+@_deprecated(msg="scipy.misc.central_diff_weights is deprecated in "
+                 "SciPy v1.10.0; and will be completely removed in "
+                 "SciPy v2.0.0. You may consider using "
```

Comment:
This should say `v1.12.0` just like the functions that were moved to `datasets`.

Reply:
And same comment for `derivative`.

Reply:
Done